netplay/createroom.cpp

```diff
@@ -107,6 +109,7 @@ void CreateRoom::onFinished(int)
     (*CoreDoCommand)(M64CMD_ROM_CLOSE, 0, NULL);
     if (!launched && webSocket)
     {
+        disconnect(webSocket, SIGNAL(error(QAbstractSocket::SocketError)), this, SLOT(handleConnectionError(QAbstractSocket::SocketError)));
```

Comment:
You don't need to do this, when the webSocket is deleted it'll do this as part of destroying the object

Reply:
Without this, I was seeing errors because calling `webSocket->close()` and `webSocket->deleteLater()` were resulting in the error signal firing.

Reply:
ok fair enough